llvm/test/MC/Disassembler/AMDGPU/gfx1150_asm_salu_float.txt

```diff
@@ -0,0 +1,2527 @@
+# RUN: llvm-mc -arch=amdgcn -mcpu=gfx1150 -disassemble -show-encoding < %s | FileCheck -check-prefixes=GFX1150 %s
```

Comment:
It looks like this was added with the wrong file name, `_asm_` instead of `_dasm_`?

Reply:
https://github.com/llvm/llvm-project/commit/0219bd3992a1b9c0ae254e3884c6da7880472fa6

Reply:
Thanks!